docs/deployment/05-foundationdb-installation.mdx

```diff
@@ -142,7 +142,7 @@ execute this to init a database
 ```sh
 configure new single ssd
 ```
-Next, execute this to from 2 other nodes to a cluster, replace the address with your machine address 
+Next, set all three nodes as coordinators, replace the address with your machine address
```

Comment:
but this is not obvious to users which place to execute the below commands

Reply:
use should execute the command on node1, not 'other 2 nodes', the original is wrong